### PR TITLE
Add Thanos Extractor argo overlays

### PR DESCRIPTION
**Pull request by gmfrasca**

### CODE REVIEW

File: argo-events/overlays/dh-prod-thanos-extractor/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: dh-prod-thanos-extractor
```
Comment (anishasthana) [inline argo-events/overlays/dh-prod-thanos-extractor/kustomization.yaml:4]: @HumairAK we don't need to specify namespace here do we?

Comment (gmfrasca) [reply argo-events/overlays/dh-prod-thanos-extractor/kustomization.yaml:4]: fwiw it is specified in every other overlays' `kustomization.yaml` files (at least for argo)

**Comment by gmfrasca** [comment]
(rebased off of AICoE/production)

File: argo-events/overlays/dh-prod-thanos-extractor/kustomization.yaml (hunk 1 above)
Comment (HumairAK) [reply argo-events/overlays/dh-prod-thanos-extractor/kustomization.yaml:4]: @anishasthana looks fine to me

**Review by anishasthana — APPROVED** [review]

**Review by HumairAK — APPROVED** [review]

**Pull request merged**